doc/Settings.md

```diff
@@ -107,3 +107,13 @@ While work is in progress for import, the command is hidden behind a feature tog
        "import": true
    },
 ```
+
+### restSource
+
+While work is in progress for rest source support, the command is hidden behind a feature toggle. One can enable it as below:
```

Comment:
>command [](start = 55, length = 7)

feature is hidden behind a toggle.

Might also want to point out that enabling this will not change how anything works, just allow for testing of the REST source.